src/main/resources/anniv5Resources/localization/eng/bladestormpack/Relicstrings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "${ModID}:FlyingDutchman": {
+    "NAME": "Flying Dutchman",
+    "FLAVOR": "You'd swear it glows and hovers when the weather's real bad.",
+    "DESCRIPTIONS": [
+      "Each turn, after playing #b",
+        " Attack, the next one  grants #b",
```

Comment:
Very minor but there are two spaces after "one" in this line and the line below. Should only be one space.